src/esp/nav/PathFinder.cpp

```diff
@@ -530,6 +535,8 @@ bool PathFinder::build(const NavMeshSettings& bs,
     }
   }
 
+  removeZeroAreaPolys();
```

Comment:
Maybe add one-line to the comment message mentioning this bug fix. It's nice have a list of bug fixes in the commit message. Can be handy later on when searching through 'git log'.